src/python/datastore/corpus_tagging.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for managing corpus tags."""
+
+from datastore import data_types
+from datastore import ndb_utils
+
+
+def get_fuzz_target_tag(fuzz_target_name):
+  """Get all the tag of a given fuzz target."""
+  query = data_types.CorpusTag().query()
+  query = query.filter(
+      data_types.CorpusTag.fuzz_target_name == fuzz_target_name)
+  return ndb_utils.get_all_from_query(query)
```

Comment:
don't use get_all_from_query. it's deprecated and will be removed soon.

Reply:
Thanks for the heads up. I replaced it with just list(query) is this sufficient, or is there a way to do this that is better practice?

Reply:
You can probably skip even the list() .The previous behaviour of ndb_utils.get_all_from_query is to return a generator anyway.